stock_picking_transfer_lot_autoassign/models/stock_picking_type.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    avoid_internal_assignment = fields.Boolean(
```

Comment:
I find this field name confusing, especially since it widely differs from the displayed name. 

Can we take the opportunity of this migration to rename it to "avoid_auto_assignment"?

Reply:
I agree..